cfgov/jinja2/v1/_includes/organisms/filterable-list-controls.html

```diff
@@ -181,7 +181,8 @@
 
 {% macro render(controls, form, index) %}
     <div class="o-filterable-list-controls">
-        {% if request.GET %}
+        {% set has_active_filters = page.has_active_filters(request) %}
```

Comment:
Could you pass in `page.has_active_filters(request.GET)`? Will the method ever need to do anything else aside from getting the querydict from the request?